### ISSUE TITLE
Problem using phantomjs with selenium

### ISSUE THREAD

**Issue by cpsievert** [ropensci/RSelenium#38]
Taking the example from the [headless browsing vignette](http://cran.r-project.org/web/packages/RSelenium/vignettes/RSelenium-headless.html#id1b) gives me an error:

```r
require(RSelenium)
RSelenium::startServer()
remDr <- remoteDriver(browserName = "phantomjs")
remDr$open()


[1] "Connecting to remote server"
Error: 	 Summary: UnknownError
 	 Detail: An unknown server-side error occurred while processing the command.
 	 class: java.lang.ClassNotFoundException
```

Oddly enough, if I change "phantomjs" to "phantom", I can connect:

```r
dr <- remoteDriver(browserName = "phantom")
dr$open()


[1] "Connecting to remote server"
$browserName
[1] "htmlunit"

$javascriptEnabled
[1] TRUE

$version
NULL

$webdriver.remote.sessionid
[1] "fbb1b432-a78b-4d73-9141-01a9499bb5c0"

$platform
[1] "MAC"

$cssSelectorsEnabled
[1] TRUE

$id
[1] "fbb1b432-a78b-4d73-9141-01a9499bb5c0"
```

However, if I navigate somewhere and try to take a screenshot, I get an error:

```r
dr$navigate("https://www.google.com/")
dr$screenshot(display = TRUE)

Error: 	 Summary: UnknownError
 	 Detail: An unknown server-side error occurred while processing the command.
 	 class: java.lang.ClassCastException
```

FYI, phantomjs is in my path:

```r
Sys.which("phantomjs")
                 phantomjs 
"/usr/local/bin/phantomjs" 
```

And here is my `sessionInfo()`:

```r
R version 3.1.2 (2014-10-31)
Platform: x86_64-apple-darwin13.4.0 (64-bit)

locale:
[1] en_US.UTF-8/en_US.UTF-8/en_US.UTF-8/C/en_US.UTF-8/en_US.UTF-8

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

other attached packages:
[1] RSelenium_1.3.5 XML_3.98-1.1    RJSONIO_1.3-0   RCurl_1.95-4.5  bitops_1.0-6   
[6] devtools_1.7.0 

loaded via a namespace (and not attached):
[1] caTools_1.17.1 tools_3.1.2   
```

**Comment by johndharrison** [comment]
What is the result of:

```
Sys.which("java")
```

**Comment by cpsievert** [comment]
```r
Sys.which("java")
           java 
"/usr/bin/java" 
```

**Comment by johndharrison** [comment]
If you are using a recent version of Selenium Server it is not possible to use phantomjs via the Selenium Server at this point in time see #30. This is a known issue with ghostdriver. You can drive phantomjs directly using the `phantom` utility function or use an older version of Selenium Server.

**Comment by cpsievert** [comment]
That's a bummer. I'll just use `phantom()` then, thanks John!

**Comment by johndharrison** [comment]
This issue is fixed as of phantomjs 2.0 and selenium server 2.45.0 see #30
